images/atomic-registry-quickstart/run.sh

```diff
@@ -26,7 +26,7 @@ echo "Starting registry services..."
 
 set -x
 
-$CMD oadm registry --credentials /etc/origin/master/openshift-registry.kubeconfig
+$CMD oadm registry
```

Comment:
@aweiteka can you make the atomic example changes in your PR, and I'll remove them from this one, then?